src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -47,6 +48,13 @@ class FileService(
     fileRepository.getFilesWithPassedAntivirus(consignmentId).map(r => Files(r.map(_.fileid)))
   }
 
+  def getFile(fileId: UUID): Future[FileDetails] = {
+    fileRepository.getFileIfPassedAntivirus(fileId).map(r => {
```

Comment:
Do we always want to apply this filter? It seems like it would be clearer to make it an explicit filter in the query.

Also, do we even need to do this filtering any more? Or did it come from a point when we thought we'd continue with the transfer but skip the file that failed antivirus? Now we're stopping the user from continuing with the consignment.

Reply:
Was following what was done for the existing `getFiles` function which applies the anti-virus check.